packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -94,10 +102,10 @@ done
 %defattr(-,root,root,-)
 %doc README.*
 %license LICENSE
-%dir %{_sysconfdir}/%{name}
+%dir %{_root_sysconfdir}/%{name}
 %config(noreplace) %{_sysconfdir}/%{name}/custom-hiera.yaml
-%dir %{_sysconfdir}/%{name}/scenarios.d
-%{_sysconfdir}/%{name}/scenarios.d/foreman.migrations
+%dir %{_root_sysconfdir}/%{name}/scenarios.d
+%{_root_sysconfdir}/%{name}/scenarios.d/foreman.migrations
 %config(noreplace) %attr(600, root, root) %{_sysconfdir}/%{name}/scenarios.d/foreman.yaml
```

Comment:
These need to be `_root_sysconfdir` as well. Otherwise I wonder if we need these root_* macros at all.